### PR TITLE
[Issue 79] - Init FingerprintRegistration component

### PR DESCRIPTION
**Pull request by natesulat**

### CODE REVIEW

File: create-ssirius-standalone/template/src/componentMap.ts [hunk 1]
```diff
@@ -103,7 +106,46 @@ const component_map = {
...
         }
+    },
+    fpRegistration: {
+        component: FingerprintRegistration,
+        props: {
+            getFingerprint: async function() {
+                const {data} = await axios.get('http://localhost:9907/EKYC/Fingerprint');
+                return data.ImageBase64;
+            },
+            register: async function(fingerprints: FingerprintMap, dependencyData: any) {
```
Comment (natesulat) [inline create-ssirius-standalone/template/src/componentMap.ts:117]: @jeffk-kiva If you could just take a look at this function and let me know if this is how the FP registration is supposed to work, that would be wonderful. I attempted to model it off of https://github.com/kiva/protocol-integration-tests/blob/main/test/v2.remote.employee.guardian.fingerprint.e2e-spec.ts#L91

File: create-ssirius-standalone/template/src/preBuilt/FingerprintRegistration/interfaces/FingerprintRegistrationInterfaces.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+export interface FingerprintCapture {
+    image: string;
+    captured: Date;
+}
+
+interface DependencyConfig {
+    [index: string]: string[]
+}
+
+export type fpIndex = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10;
```
Comment (jeffk-kiva) [inline create-ssirius-standalone/template/src/preBuilt/FingerprintRegistration/interfaces/FingerprintRegistrationInterfaces.ts:44]: Please capitalize this. I know I shouldn't be commenting on style things, but this should really have the same naming style as classes and interfaces.

File: create-ssirius-standalone/template/src/componentMap.ts (hunk 1 above)
Comment (jeffk-kiva) [reply create-ssirius-standalone/template/src/componentMap.ts:117]: I think this looks right!

File: create-ssirius-standalone/template/src/preBuilt/FingerprintRegistration/interfaces/FingerprintRegistrationInterfaces.ts (hunk 2 above)
Comment (natesulat) [reply create-ssirius-standalone/template/src/preBuilt/FingerprintRegistration/interfaces/FingerprintRegistrationInterfaces.ts:44]: Nah, comment on style stuff

**Review by jeffk-kiva — APPROVED** [review]

**Pull request merged**